controllers/kustomization_gc.go

```diff
@@ -138,6 +142,21 @@ func (kgc *KustomizeGarbageCollector) isStale(obj unstructured.Unstructured) boo
 	return kgc.newChecksum == "" || itemChecksum != kgc.newChecksum
 }
 
+func (kgc *KustomizeGarbageCollector) shouldSkip(obj unstructured.Unstructured) bool {
+	key := fmt.Sprintf("%s/prune", kustomizev1.GroupVersion.Group)
+	val := "disabled"
+
+	if obj.GetLabels()[key] == val {
+		return true
+	}
+
+	if obj.GetAnnotations()[key] == val {
+		return true
+	}
+
+	return false
```

Comment:
Can't this simply be reduced to the following?

```suggestion
	return obj.GetLabels()[key] == val || obj.GetAnnotations()[key] == val
```